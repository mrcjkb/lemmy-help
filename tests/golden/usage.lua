local U = {}

---Prints a message
---@param msg string Message
---@usage lua [[
---require('module.U').sum(10, 5)
---@usage ]]
function U.echo(msg)
    print(msg)
end

---Add a number to 10
---@param this number A number
---@usage `require('module.U').sum(5)`
function U.sum(this)
    print(10 + this)
end

return U
