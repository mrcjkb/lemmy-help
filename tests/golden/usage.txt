U.echo({msg})                                                           *U.echo*
    Prints a message

    Parameters: ~
        {msg}  (string)  Message

    Usage: ~
>lua
        require('module.U').sum(10, 5)
<


U.sum({this})                                                            *U.sum*
    Add a number to 10

    Parameters: ~
        {this}  (number)  A number

    Usage: ~
>lua
        require('module.U').sum(5)
<



